packages/ant-component-mapper/src/files/field-array.js

```diff
@@ -195,15 +251,46 @@ DynamicArray.propTypes = {
   minItems: PropTypes.number,
   maxItems: PropTypes.number,
   noItemsMessage: PropTypes.node,
+  buttonLabels: PropTypes.object,
+  // customization props
   FormItemProps: PropTypes.object,
-  buttonLabels: PropTypes.object
+  ArrayItemProps: PropTypes.object,
+  FieldsContainerProps: PropTypes.object,
+  RemoveContainerProps: PropTypes.object,
+  RemoveButtonProps: PropTypes.object,
+  FieldArrayRowProps: PropTypes.object,
+  FieldArrayRowCol: PropTypes.object,
+  FieldArrayHeaderProps: PropTypes.object,
+  FieldArrayLabelProps: PropTypes.object,
+  FieldArrayButtonsProps: PropTypes.object,
+  UndoButtonProps: PropTypes.object,
+  RedoButtonProps: PropTypes.object,
+  AddButtonProps: PropTypes.object,
+  FieldArrayDescriptionProps: PropTypes.object,
+  NoItemsMessageProps: PropTypes.object,
+  ErrorMessageProps: PropTypes.object
 };
 
 DynamicArray.defaultProps = {
   maxItems: Infinity,
   minItems: 0,
   noItemsMessage: 'No items added',
-  FormItemProps: {}
+  FormItemProps: {},
```

Comment:
As there are no props/classNames merging with these, do we really need all these defaults?

Reply:
There is no difference between `undefined` and `{}` in the spread operators true, but this will help with the docs props generator I have in plan.